cookbooks/aws-parallelcluster-slurm/recipes/install_slurm.rb

```diff
@@ -50,6 +50,25 @@
   shell '/bin/bash'
 end
 
+# Setup slurmdbd group
+group node['cluster']['slurm']['dbdgroup'] do
+  comment 'slurm group'
+  gid node['cluster']['slurm']['dbdgroup_id']
+  system true
+end
+
+# Setup slurmdbd user
+user node['cluster']['slurm']['dbduser'] do
+  comment 'slurmdbd user'
+  uid node['cluster']['slurm']['dbduser_id']
+  gid node['cluster']['slurm']['dbdgroup_id']
+  # home is mounted from the head node
+  manage_home ['HeadNode', nil].include?(node['cluster']['node_type'])
+  home "/home/#{node['cluster']['slurm']['dbduser']}"
```

Comment:
can we avoid creating the home directory for this user? So that we do no add new deps on the /home if not necessary

Reply:
I would leave this to a separate PR where we remove the homes for system users (`slurm` has a home directory too).

Reply:
We will remove the home directory for `slurmdbd`.

Reply:
Done.